syntax="proto3" ;

message PersonMessage{
    int32 id =1;
    string first_name = 2;
    string last_name = 3;
    string company_name = 4;
}


message PersonsMessage {
    repeated PersonMessage persons =1;
}
message PersonIdMessage {
    int32 id =1;
}

message Empty{

}
service PersonsService{
    rpc GetAllPersons(Empty) returns (PersonsMessage);
    rpc GetPerson(PersonIdMessage) returns (PersonMessage);
}